aws/templates/solution/solution_ec2.ftl

```diff
@@ -151,26 +232,32 @@
                                                 "Fn::Join" : [
                                                     "",
                                                     [
-                                                        "#!/bin/bash\n",
-                                                        "echo \"cot:request="       + requestReference       + "\"\n",
-                                                        "echo \"cot:configuration=" + configurationReference + "\"\n",
-                                                        "echo \"cot:accountRegion=" + accountRegionId        + "\"\n",
-                                                        "echo \"cot:tenant="        + tenantId               + "\"\n",
-                                                        "echo \"cot:account="       + accountId              + "\"\n",
-                                                        "echo \"cot:product="       + productId              + "\"\n",
-                                                        "echo \"cot:region="        + regionId               + "\"\n",
-                                                        "echo \"cot:segment="       + segmentId              + "\"\n",
-                                                        "echo \"cot:environment="   + environmentId          + "\"\n",
-                                                        "echo \"cot:tier="          + tierId                 + "\"\n",
-                                                        "echo \"cot:component="     + componentId            + "\"\n",
-                                                        "echo \"cot:zone="          + zone.Id                + "\"\n",
-                                                        "echo \"cot:name="          + formatName(ec2FullName, zone) + "\"\n",
-                                                        "echo \"cot:role="          + component.Role         + "\"\n",
-                                                        "echo \"cot:credentials="   + credentialsBucket      + "\"\n",
-                                                        "echo \"cot:code="          + codeBucket             + "\"\n",
-                                                        "echo \"cot:logs="          + operationsBucket       + "\"\n",
-                                                        "echo \"cot:backups="       + dataBucket             + "\"\n"
-                                                    ]
+                                                        "#!/bin/bash\\n",
```

Comment:
these need adjustment. there was a change to getJSON so that you don't need all the extra backslashes

Reply:
Fixed up